src/styles/blocks/catalog.scss

```
@@ -17,9 +17,16 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition-duration: 300ms, 300ms;
+    transition-property: color, transform;
```

you can do like this
t
```suggestion
    transition: color 300ms, transform 300ms;
```

---

Also, would be better if you create a variable for transitionDuration